.github/workflows/CI.yml

```diff
@@ -78,3 +78,22 @@ jobs:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
       - run: bundle exec rake
+
+  TruffleRuby:
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [ '22.2.0' ]
```

Comment:
I think it would be better to test the latest truffleruby release, so just `truffleruby` here and `ruby-version: ${{ matrix.ruby }}` below (or just no matrix).